maestro/scripts/parse-taps.py

```
@@ -0,0 +1,93 @@
+        )
+
+    out.append("")
+    out.append("# Maestro snippets:")
+    for i, (t, px, py) in enumerate(taps, 1):
+        if i > 1:
+            delay_ms = int((t - taps[i - 2][0]) * 1000)
+            if delay_ms >= 500:
+                seconds = max(1, round(delay_ms / 1000))
+                out.append(f"- runFlow: ../subflows/wait_{seconds}s.yaml")
+        cx, cy = pct(px, py)
+        out.append(f"- tapOn:")
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)